wagtail/contrib/search_promotions/templates/wagtailsearchpromotions/includes/searchpromotion_form.html

```diff
@@ -22,8 +22,15 @@
             </ul>
         {% endif %}
 
+        {% trans "Use an internal Page for this Promotion" %}
         {% include "wagtailadmin/shared/field.html" with field=form.page only %}
+
+        {% trans "Alternatively, use an external link for this Promotion" %}
+        {% include "wagtailadmin/shared/field.html" with field=form.external_link_url only %}
+        {% include "wagtailadmin/shared/field.html" with field=form.external_link_title only %}
+
```

Comment:
```suggestion
        {% trans "Alternatively, use an external link for this Promotion" %}
        {% include "wagtailadmin/shared/field.html" with field=form.external_link_url only %}
        {% include "wagtailadmin/shared/field.html" with field=form.external_link_title only %}
```

For now, no need for the extra added whitespace.